content/rc/cloud-integrations/aws-marketplace/aws-bedrock.md

```diff
@@ -0,0 +1,107 @@
+    | 1,000,000 kb                  | 13.5 Gb                              | 27 Gb                          |
+    | 10,000,000 kb                 | 135 Gb                             | 270 Gb                         |
+
+    For more information on sizing, see the [Bedrock integration blog post](https://redis.com/blog/amazon-bedrock-integration-with-redis-enterprise#right-size-your-database-for-amazon-bedrock).
+
+### Get TLS certificates {#get-certs}
+
+1. Enable [Transport Layer Security (TLS)]({{<relref "/rc/security/database-security/tls-ssl#enable-tls">}}) for your database. Make sure to check **TLS client authentication** to require client authentication. Download the client certificates before saving your changes.
+
+1. If you do not have the Redis Cloud server certificates, [download them]({{<relref "/rc/security/database-security/tls-ssl#download-certificates">}}) from the admin console.
+
+### Store database credentials in AWS secrets manager {#store-secret}
+
+In the [AWS Management Console](https://console.aws.amazon.com/), use the **Services** menu to locate and select **Security, Identity, and Compliance** > **Secrets Manager**. [Create a secret](https://docs.aws.amazon.com/secretsmanager/latest/userguide/create_secret.html) with the following fields:
+
+- `username`: Database username
+- `password`: Database password
+- `serverCertificate`: Contents of the [server certificate]({{<relref "/rc/security/database-security/tls-ssl#download-certificates">}}) (`redis_ca.pem`)
+- `clientCertificate`: Contents of the client certificate (`redis_user.crt`)
+- `clientPrivateKey`: Contents of the client private key (`redis_user_private.key`)
+
+Once you store this secret, you can view and copy the [Amazon Resource Name (ARN)](https://docs.aws.amazon.com/secretsmanager/latest/userguide/reference_iam-permissions.html#iam-resources) of your secret on the secret details page. 
+
+### Create a vector index in your database {#create-vector-index}
+
+Once your database is set up, you need to create an index with a vector field as your knowledge base for Amazon Bedrock.
+
+#### [RedisInsight](https://redis.io/docs/ui/insight/)
+
+RedisInsight is a free Redis GUI that allows you to visualize and optimize your data in Redis. 
+
+Follow the steps in [this dedicated guide](https://github.com/RedisVentures/aws-redis-bedrock-stack/blob/main/docs/vector-index-creation.md) to create your vector index with RedisInsight.
+
+#### [`redis-cli`]({{<relref "/rs/references/cli-utilities/redis-cli">}})
+
+The `redis-cli` command-line utility lets you connect and run Redis commands directly from the command line. To gain access to `redis-cli`, you can [install Redis](https://redis.io/docs/getting-started/).
+
+Follow the instructions to [connect to your database with TLS certificates]({{<relref "/rc/security/database-security/tls-ssl#connect-with-the-redis-cli">}}).
+
```

Comment:
Seems like there's some missing text for the CLI example shown. Maybe something like, "Here's an example of <squidblat>:"